evm/src/main/java/org/hyperledger/besu/evm/gascalculator/EIP1153GasCalculator.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Besu Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.evm.gascalculator;
+
+public class EIP1153GasCalculator extends LondonGasCalculator {
+
+  private static final long TLOAD_GAS = WARM_STORAGE_READ_COST;
+  private static final long TSTORE_GAS = WARM_STORAGE_READ_COST;
+
+  public EIP1153GasCalculator() {}
+
+  // EIP-1153
+  @Override
+  public long getTransientLoadOperationGasCost() {
+    return TLOAD_GAS;
```

Comment:
Why not always set the cost to TLOAD_GAS?  Why update it with a fresh gas calculaotr?

Reply:
Since it has to be included in a gas calculator, I was thinking this would be renamed to the hardfork that this EIP is ultimately included in. Alternatively I can add it to the `LondonGasCalculator` now. WDYT?

Reply:
TSTORE/TLOAD hasn't been "EFIed" for any hard fork, so it would be inappropriate to put it in a named hardfork. (such as in this list - https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/shanghai.md).

Instead I think we need to activate EIPs by number and make EVM decisions based on that config.  In the config files it might look like this - https://github.com/shemnon/besu/tree/eips_in_config

It still would need to be wired into the EVM config, both as integrated in Besu and as a standalone library.

Reply:
> Instead I think we need to activate EIPs by number and make EVM decisions based on that config.

Is this practice already in place or are you suggesting that's how it should be handled in the future?